docs/framework/guides/contributing/code-style.md

```diff
@@ -447,7 +447,7 @@ You can use ES6 getters to simplify class API:
 class Position {
 	// ...
 	get offset() {
-		return this.path[ this.path.length - 1 ];
+		return this.path[ this.path.length * 1 ];
```

Comment:
Not sure how this has been changed unexpectedly from `this.path.length - 1` to `this.path.length * 1`, but I will fix that.